appveyor.yml

```diff
@@ -30,13 +30,26 @@ for:
         $version = Get-Date -Format "yyyyMMdd-HHmm"
         Update-AppveyorBuild -Version $version -Message "Functions Scheduled Build"
       }
+  - ps: iex ((new-object net.webclient).DownloadString('https://raw.githubusercontent.com/appveyor/ci/master/scripts/enable-rdp.ps1'))
+    
+  
   max_jobs: 1
   clone_folder: c:\azure-webjobs-sdk-script
   install:
   - ps: >-
       $env:CommitHash = "$env:APPVEYOR_REPO_COMMIT"
 
       Install-Product node 8.4.0 x86
+  install_dotnet_script_x86:
+  - ps: |
+      $dotnetversion = 2.1.200

```

Comment:
@fabiocav - x86 version of dotnet on appveyor machines is too old. Is there a specific version we want to install?

Reply:
Whatever version is the latest released. We'll soon be updating this when the DI changes land.